src/modules/45-dashboards/pages/home/HomePage.tsx

```diff
@@ -119,7 +119,11 @@ const HomePage: React.FC = () => {
   const [selectedTags, setCheckboxFilter] = useQueryParamsState('tags', DEFAULT_FILTER)
   const [layoutView, setLayoutView] = useQueryParamsState('view', DashboardLayoutViews.GRID)
 
-  const { includeBreadcrumbs } = useDashboardsContext()
+  const { editableFolders, includeBreadcrumbs } = useDashboardsContext()
+
+  const hasEditableFolders: boolean = React.useMemo(() => {
+    return editableFolders ? editableFolders.length > 0 : false
+  }, [editableFolders])
```

Comment:
Same here. For simple things, the operation is often faster than memoisation and lookup.
```suggestion
  const hasEditableFolders: boolean = !!editableFolders?.length
```